src/lib/utils/gasLimitHelper.ts

```diff
@@ -0,0 +1,25 @@
+import { GasGroup, MAX_GAS_PER_GROUP } from '~app/common/config/gasLimits';
+import { DEVELOPER_FLAGS, getLocalStorageFlagValue } from '~lib/utils/developerHelper';
+
+// const USE_HARDCODED_GAS_LIMIT = 'USE_HARDCODED_GAS_LIMIT';
+
+const calculateTenPercentIncrease = (value: number): number | null =>  {
+      if ( getLocalStorageFlagValue(DEVELOPER_FLAGS.USE_HARDCODED_GAS_LIMIT) === 1) {
+          return Math.floor(value * 1.2);
+      }
+    return null;
+};
```

Comment:
name is "calculateTenPercentIncrease"
but you are doing 20% -> value * 1.2

Reply:
done